torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -20,15 +20,15 @@ def horizontal_flip_bounding_box(bounding_box: torch.Tensor, *, image_size: Tupl
         old_format=BoundingBoxFormat.XYXY,
         new_format=BoundingBoxFormat.XYWH,
     ).unbind(-1)
-    x = image_size[1] - (x + w)
+    x = image_size[1] - (x + w)  # Why not avoid formatting and do straight `boxes[:, [0, 2]] = width - boxes[:, [2, 0]]`?
     return convert_bounding_box_format(
         torch.stack((x, y, w, h), dim=-1),
         old_format=BoundingBoxFormat.XYWH,
         new_format=BoundingBoxFormat.XYXY,
     )
 
 
-_resize_image = _F.resize
+_resize_image = _F.resize  # How about videos? The low level transfroms supports it
```

Comment:
Right now there is custom feature type for videos and the tentatively plan is to treat videos as `Image`'s with an extra batch dimensions cc @bjuncek. Since the kernel should be agnostic to batch dimensions, they are automatically supported by all image transforms.

Reply:
We should finalize this plan soon. We got outstanding work with Videos so we need to be able to have an idea of how this works for them. Once the plan is finalized, we will be able to parallelize the work of porting all transforms concurrently.

I agree about the underlying kernels being agnostic, sounds good.

Reply:
For reference, the initial support for videos happens in #5335. @bjuncek is checking with other researchers working on videos if the API is sufficient or if we are missing something important.

Reply:
Annoyingly, I still haven't gotten a reply from any of them.
@datumbox , maybe you could help me offline in order to make this discussion happen?

> tentatively plan is to treat videos as Image's with an extra batch dimensions

Is there a downside to creating a `video` feature class that for the time being implement just this, but would be possible to extend with stuff like audio or cc-data or encoded bytes if support for this will be needed?

Reply:
@bjuncek ping me offline with the names of the people you try to talk to to see if I can help.

When we say "extra branch dimensions", could you clarify the proposal?

Reply:
I don't think there is real downside other than being more verbose in our transforms implementations. For example, we would need to have `resize_video` that simply defers to `resize_image`. This has to happen for every single image / video kernel.